locales/locales/en.json

```diff
@@ -1629,6 +1629,27 @@
 	"accountSwitcherReloadOtherTabsDesc": {
 		"message": "After switching accounts, automatically reload other tabs."
 	},
+	"accountSwitcherAutoSwitchOnPostTitle": {
+		"message": "Automatically Switch To Post OP"
+	},
+	"accountSwitcherAutoSwitchOnPostDesc": {
+		"message": "Automatically switch to alternative account if it's the OP of the post you are on."
+	},
+	"accountSwitcherAutoSwitchOnCommentTitle": {
+		"message": "Automatically Switch To Comment OP"
+	},
+	"accountSwitcherAutoSwitchOnCommentDesc": {
+		"message": "Automatically switch to alternative account if a comment from one of your alternative accounts is on the page"
+	},
+	"accountSwitcherAutoSwitchNotificationTitle": {
+		"message": "Account Switched"
+	},
+	"accountSwitcherAutoSwitchPostDesc": {
+		"message": "Your account has been switched to <b>$1</b> because it is the OP of this post"
+	},
+	"accountSwitcherAutoSwitchCommentDesc": {
+		"message": "Your account has been switched to <b>$1</b> because a comment by it was found on this post."
```

Comment:
How about..

`"You have been automatically switched to <b>u/$1</b> because this account submitted a comment on this post."